mods/seasonal/moves.js

```diff
@@ -533,6 +562,33 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Water",
 	},
+	// Kay
+	inked: {
+		accuracy: 100,
+		basePower: 80,
+		category: "Physical",
+		desc: "Lowers the user's Defense and Special Defense by 1 stage.",
+		shortDesc: "Lowers the user's Defense and Sp. Def by 1.",
+		id: "inked",
+		name: "Inked",
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Psywave', target);
+			this.add('-anim', source, 'Octazooka', target);
+		},
+		self: {
+			boosts: {
+				def: -1,
+				spd: -1,
```

Comment:
It's supposed to lower speed one stage as well

Reply:
Done